### PR TITLE
Cat 2748 update catalog details

### PR DESCRIPTION
**Pull request by suyebmendix**

### CODE REVIEW

File: content/en/docs/catalog/manage/search.md [hunk 1]
```diff
@@ -436,15 +438,15 @@ The data source URI is the location of the service contract of the data source,
...
+For a selected data source, you can click **Download** to download the service contract that is located at the data source endpoint. A ZIP file that includes the all the files that make up the full contract is generated and downloaded.
 
 The resulting ZIP file is named `DataHub_<service_name>_<service_version>_<technology>.zip` where the string `<technology>` identifies the service protocol.
 
 Here is an example:
 
 {{< figure src="/attachments/catalog/search/download_example.png" alt="download example" >}}
 
-When you click **Download Contract**, the following file is downloaded: `DataHub_SAP_Intelligence_1.0_OData4.zip`. This ZIP file has the folder `DataHub_SAP_Intelligence_1.0_OData4`, which contains the all the metadata files that define the service.
+When you click **Download**, the following file is downloaded: `DataHub_SAP_Intelligence_1.0_OData4.zip`. This ZIP file has the folder `DataHub_SAP_Intelligence_1.0_OData4`, which contains the all the metadata files that define the service.
```
Comment (BananiMendix) [inline content/en/docs/catalog/manage/search.md]: Found an existing issue. There is an extra the, should be "which contains all the metadata files"

File: content/en/docs/catalog/manage/curate.md [hunk 2]
```diff
@@ -154,30 +157,26 @@ You can curate data sources to add a Catalog description and add tags.
...
 
-1. In the **Curation Bar**, click **Edit** > **Edit Metadata** > **Service Details** to display the **Service Metadata** box:
+1. In the **Curation Option**, click **Edit** to display the **Edit** box:
 
     {{< figure src="/attachments/catalog/curate/service-metadata-box.png" alt="service metadata" >}}
 
-2. The description from the OData service metadata is displayed in the **Contract Description**. This cannot be changed, as it is part of the OData service definition. You can provide an alternative description by toggling **Use the catalog description** to display the **Catalog Description** box:
-
-    {{< figure src="/attachments/catalog/curate/service-metadata-description.png" alt="edit description" >}}
+2. The description from the service metadata is displayed in the **Description provided by the contract**. This cannot be changed, as it is part of the service definition. You can provide an alternative description in **Description** box.
```
Comment (BananiMendix) [inline content/en/docs/catalog/manage/curate.md]: alternative description in the **Description** box.

File: content/en/docs/catalog/manage/search.md [hunk 3]
```diff
@@ -289,15 +289,12 @@ Enter a search string in the **Search** area with a minimum of 3 alphanumeric ch
 
 #### 3.4.2 Filters {#filter}
 
-You can filter search results by environment type. The **Production** environment filter is active by default.
+You can filter search results by 
+* Environment type. The **Production** environment filter is active by default.
```
Comment (BananiMendix) [inline content/en/docs/catalog/manage/search.md]: Is still the Production environment selected by default ?

File: content/en/docs/catalog/manage/curate.md [hunk 4]
```diff
@@ -52,24 +52,28 @@ When a data source is selected in the **Search Results**, you can do the followi
 * [Discoverable/Validated](#discoverability) – set the **Discoverable** and **Validated** settings for the data source from the drop-down menu
```
Comment (BananiMendix) [inline content/en/docs/catalog/manage/curate.md]: This part needs to be updated as `Discoverable` setting is on the Edit page and on the My Asstes/Company Assets list and `Validated`  setting is on the Edit page.

File: content/en/docs/catalog/manage/search.md (hunk 3 above)
Comment (suyebmendix) [reply content/en/docs/catalog/manage/search.md]: Yes

File: content/en/docs/catalog/manage/curate.md (hunk 2 above)
Comment (suyebmendix) [reply content/en/docs/catalog/manage/curate.md]: Changed

File: content/en/docs/catalog/manage/search.md (hunk 1 above)
Comment (suyebmendix) [reply content/en/docs/catalog/manage/search.md]: changed

File: content/en/docs/catalog/manage/curate.md (hunk 4 above)
Comment (suyebmendix) [reply content/en/docs/catalog/manage/curate.md]: Changed

File: content/en/docs/catalog/manage/curate.md [hunk 5]
```diff
@@ -90,9 +90,9 @@ Curators and Mendix Admins can manage the lists of custom owners as part of thei
...
 
-To change the **Business Owner** or **Technical Owner**, from the **Application** tab, click in the name field of the **Business** or **Technical** owner. You can start typing or select from the drop-down list of names. Mendix Platform users will be displayed with their avatar and custom owners will be shown with an avatar that displays their initials.
+To change the **Business Owner** or **Technical Owner** from the **Application** tab, click in the name field of the **Business** or **Technical** owner. You can start typing or select a name from the drop-down list. Mendix Platform users will be displayed with their avatar and custom owners will be shown with an avatar that displays their initials.
 
-The **Technical Owner** can only be changed in **Control Center**, not within the **Catalog**. See the [Custom Owners](/developerportal/control-center/catalog-admin/#custom-owners) section of *Catalog Administration*.
+The **Technical Owner** can only be changed in **Control Center**, not within the Catalog. For more information, see the [Custom Owners](/developerportal/control-center/catalog-admin/#custom-owners) section of *Catalog Administration*.
```
Comment (quinntracy) [inline content/en/docs/catalog/manage/curate.md]: @BananiMendix Is there a link to this **Control Center** page? Or is this the **Owners** tab on the Curate page?

File: content/en/docs/catalog/manage/curate.md [hunk 6]
```diff
@@ -112,11 +112,11 @@ To add a custom **Business Owner**, follow these steps:
 
 ### 4.2 Changing the App Icon {#application-icon}
 
-You can change the icon that is displayed for the app and the data source by uploading your own custom application icon or selecting one from the icon library. The selected icon will be displayed for all endpoints of the application and data source such as the Catalog search results and also in the Mendix Studio Pro [Integration Pane](/refguide/integration-pane/).
+You can change the icon that is displayed for the app and data source by uploading your own custom application icon or selecting one from the icon library. The selected icon will be displayed for all endpoints of the application and data source, such as the Catalog search results and in the Mendix Studio Pro [Integration pane](/refguide/integration-pane/).
```
Comment (Adam-Dupaski) [inline content/en/docs/catalog/manage/curate.md:115]: Yes, should be "pane" throughout the Catalog docs

**Pull request merged**